src/engine.rs

```diff
@@ -692,6 +702,22 @@ mod tests {
         let actual_output2: serde_json::Value =
             serde_json::from_str(&d2.data).expect("error unpacking js data");
         assert_eq!(actual_output2, expected_output);
+
+        let d3: &OutData = &od_vec[2];
+        assert_eq!(d3.out_path, None);
+        assert_eq!(d3.out_ext, Some(String::from(".hcl")));
+        assert_eq!(d3.out_prefix, None);
+        let actual_output3: serde_json::Value =
+            hcl::from_str(&d3.data).expect("error unpacking hcl data");
+        assert_eq!(actual_output3, expected_output);
+
+        let d4: &OutData = &od_vec[3];
+        assert_eq!(d4.out_path, None);
+        assert_eq!(d4.out_ext, Some(String::from(".tf")));
+        assert_eq!(d4.out_prefix, None);
+        let actual_output4: serde_json::Value =
+            hcl::from_str(&d3.data).expect("error unpacking hcl data");
+        assert_eq!(actual_output4, expected_output);
```

Comment:
I should use a different `expected_output` here so I actually validate the HCL is in the right format vs the tests passing with JSON at the moment.

Reply:
You actually need to validate the string itself, not the serde object. I didn't care much about how the output looks for yaml and json, but obv that matters here with HCL.

Reply:
I've switched to comparing `hcl::Value`'s in my latest commit, but I think your point still stands. I need to actually check the string contents.

Reply:
Checking string contents now.